benchmarks/prove/Cargo.toml

```diff
@@ -33,8 +33,9 @@ metrics.workspace = true
 [dev-dependencies]
 
 [features]
-default = ["parallel", "jemalloc", "metrics", "evm"]
```

Comment:
not sure why this was changed

Reply:
I didn't want evm in default to compile stuff faster